### PR TITLE
added Feb meeting stub

### PR DESCRIPTION
**Pull request by Kelvinrr**
The meeting tomorrow is going to be canceled, been out sick for some time and we dont have much in terms of discussion topics. 

We can use this to add topics for March.

### CODE REVIEW

File: meetings/2024-02-8.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+### Attending
+-
+
+## Agenda / Notes
+- Canceled
+
+## Discussions for next meeting
+- C++17 in ISIS
+-
```
Comment (Kelvinrr) [inline meetings/2024-02-8.md:13]: @KrisBecker Remind me of what you mentioned at the Apex meeting, we can probably get it added for March.

**Comment by thareUSGS** [comment]
Not sure I have this correct -- maybe a couple items.
1. option for short-term meeting or "pair-programming" for PRs... maybe
2. You talked about an RFC for the C++ complier version. Not sure that needs TC support.

**Comment by KrisBecker** [comment]
Yes that is what we discussed as potential topics for the next meeting.

Regarding the C++ complier issue, we would like to discuss the implications/impact of adopting the [C++17 standard](https://en.cppreference.com/w/cpp/17) for ISIS and its dependencies.

**Comment by AndrewAnnex** [comment]
I'd also like to add a point about ensuring there are additional maintainers on relevant conda-forge feedstocks exclusively serving ISIS3. I have requested being added to 2-3 repos some weeks ago and I think there should be some redundancy/bi-annual review for all ISIS dependencies to ensure things are running smoothly with dependency updates

**Comment by jlaura** [comment]
@AndrewAnnex :+1: I think that is a great discussion. Within that, perhaps we can discuss whether folks need to be contributors to the source repo or if we are at a point (I think we are) where we can have different groups, some that are contributing to the code base and some that are supporting the packaging and delivery. This might be a very tangible place where the broader community can help the core development team a lot. Thanks for bringing it up!

**Comment by KrisBecker** [comment]
Hmm, looks like ISIS was [recently](https://github.com/DOI-USGS/ISIS3/commit/968dec6e78c3b120e8aa71f1499a3973ef652faf#) updated to use the c++17 standard. However, [ALE](https://github.com/DOI-USGS/ale/blob/775ff216791349f44a643a1080818631f6b7b16e/CMakeLists.txt#L19) and [USGSCSM](https://github.com/DOI-USGS/usgscsm/blob/48c55c429ffaf7cd109d5ba0d20eb027e778cd31/CMakeLists.txt#L10) are still at c++11. [SpiceQL](https://github.com/DOI-USGS/SpiceQL/blob/bcc038492fb4be4135f596a0e46da1d6d9db8ddd/CMakeLists.txt#L10-L16) uses c++17 and c++20.

Isn't that a [problem](https://maskray.me/blog/2023-06-25-c++-standard-library-abi-compatibility)? Maybe not?

**Comment by Kelvinrr** [comment]
@KrisBecker The fact that it was already updated is a bit concerning, but it seemed to be updated specifically because we needed it to be up to date with other libraries. So it seems like it was more of a problem to be stuff in 11. We should probably get everything to at least 17 at this point.

**Review by jlaura — APPROVED** [review]

**Pull request merged**